### DISCUSSION TITLE
TreeTable. Problem with frozen.

### DISCUSSION THREAD

**Discussion by ALEX-VOIT** [primefaces/community#2517] [PrimeReact] [answered]
if the data in a cell increases the height of a non-frozen column, then the frozen one does not adapt. As far as I can see, the column with the prop frozen is placed in a separate line, therefore it cannot have the correct table behavior. I discovered the problem in my project, but easily reproduced it in the documentation by adding text to the cell.
![image](https://github.com/user-attachments/assets/32422167-6e4a-4b3b-afb2-b12472003ed3)

**Comment by melloware** [comment, marked as answer]
Issue opened: https://github.com/primefaces/primereact/issues/7059
